lib/models/LifecycleConfiguration.js

```diff
@@ -709,6 +992,40 @@ class LifecycleConfiguration {
                     Action = `<${actionName}>${Days}${Date}${DelMarker}` +
                         `</${actionName}>`;
                 }
+                if (actionName === 'NoncurrentVersionTransition') {
```

Comment:
kind of confusing that these conditionals don't match the format of the others, with setting each XML part to a variable. I don't have a strong preference for either format, but would it be possible/not a massive overhaul to sync transitions up with expiration format?

Reply:
Woops, looks like I'm too late to look at the link, but I didn't realize there could be more than one transition per rule, so nevermind 😸